bin/push-debian.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Example usage:
+#
+# bin/push-debian.sh \
+#   -c opt
+#   -v 0.2.1
+#   -p gs://istio-release/releases/0.2.1/deb
+
+function usage() {
+  echo "$0 \
+    -c <bazel config to use> \
+    -p <GCS path, e.g. gs://istio-release/releases/0.2.1/deb> \
+    -v <istio version number>"
+  exit 1
+}
+
+while getopts ":c:p:v:" arg; do
+  case ${arg} in
+    c) BAZEL_CONFIG="--config=${OPTARG}";;
+    p) GCS_PATH="${OPTARG}";;
+    v) ISTIO_VERSION="${OPTARG}";;
+    *) usage;;
+  esac
+done
+
+if [ -z "${BAZEL_CONFIG}" ] || [ -z "${ISTIO_VERSION}" ] || [ -z "${GCS_PATH}" ]; then
+  usage
+fi
+
+set -ex
+
+bazel ${BAZEL_STARTUP_ARGS} build \
+  ${BAZEL_CONFIG} --output_groups=static \
+  "//tools/deb:istio-agent"
```

Comment:
does deb rule properly handle static output group? there's a bug in bazel for doing something similar for docker, and it doesn't do the right thing.

Reply:
Removed.

Reply:
Ok, since it's urgent, that's ok, but we should consider doing a workaround in bazel with file groups in the future.

Reply:
I have no sense of the urgency.  I only added it because @rshiraram suggested this was a trivial change.  Since I'm unfamiliar with the Bazel bug or the urgency of this change I'm not going to go back and forth anymore - whoever is invested in this change can make it.

Reply:
Understood, cc'ed the right person.

Reply:
Oh, I didn't know that bazel had another bug for static output for debian files as well. :(. Will the script above work?